projects/canopy/src/lib/docs/principles/accessibility/developer.stories.mdx

```diff
@@ -0,0 +1,222 @@
+import { Meta } from '@storybook/addon-docs';
+import Feedback from '../../common/feedback.md';
+
+<Meta title="Principles/Accessibility/Developer" parameters={{ viewMode: 'docs', previewTabs: { canvas: { hidden: true } } }} />
+
+# For developers
+
+<p class="standfirst">Here we provide information on developing content to be accessible.</p>
+
+## Developing with accessibility in mind
+
+A great deal of web content can be made accessible just by making sure the correct HTML elements are used for the correct purpose at all times.
+
+To make web content and web applications more accessible to people with disabilities you can use WAI-ARIA, the Accessible Rich Internet Applications Suite. It especially helps with dynamic content and advanced user interface controls developed with Ajax, HTML, JavaScript, and related technologies.
+
+**Components from the Canopy Angular library have been built to be accessible.**
+
+### Defining the language of the document
+
+Telling the browser which language you are using in your document has many benefits. It's good for SEO and it helps third-party translation tools and browsers to identify the right language and dictionary. Defining the correct language in an HTML page helps assistive technology to choose the correct voice profile or character set.
```

Comment:
Just a suggestion:

Defining the language used in an HTML page has many benefits. It explicitly tells the browser what language you are using, thus improving SEO. Third-party translation tools and browsers can use the language information to use the correct dictionary. Additionally, assistive technologies can choose the suitable voice profile or character set.